assets/app/actions/actionCreators/alertActions.js

```diff
@@ -0,0 +1,39 @@
+import { alertActionTypes as types } from '../../constants';
+
+const authErrorType = types.AUTH_ERROR;
```

Comment:
I support dropping any "this is a bag of constants" files whenever we can.

Exposing the type right now for testing seems questionable until it has real meaning in our application or real merit in tests, but copying the "ACTION NAME" constants in the test is a bit wonky too, I agree.